packages/api-core/src/graphql/schema.graphql

```diff
@@ -34,6 +34,130 @@ enum MassUnit {
   oz
 }
 
+"Logical Operator Types for filtering"
+enum LogOpTypes{
+  "AND"
+  AND
+
+  "OR"
+  OR
+}
+
+"Relational Operator Types for filtering"
+enum RelOpTypes{
+  "Begins With used with String types"
+  beginsWith
+
+  "Ends With used with String types"
+  endsWith
+
+  "Equal"
+  eq
+
+  "Greater Than"
+  gt
+
+  "Greater Than or Equal"
+  gte
+
+  "In"
+  in
+
+  "Less Than"
+  lt
+
+  "Less Than or Equal"
+  lte
+
+  "Not Equal"
+  ne
+
+  "Not In"
+  nin
+
+  "Regex"
+  regex
+}
+
+"Flag to specify the number of levels used in Filter Search"
+enum FilterLevels {
+  "Use Filter with 1 level"
+  ONE
+
+  "Use Filter with 3 levels"
+  THREE
+
+  "Use Filter with 2 levels"
+  TWO
+}
+
+"Single Condition for filterSearch"
+input SingleConditionInput {
+  "Value to filter if it is Boolean input"
+  boolValue: Boolean
+
+  "Flag to set if the regex is case insensitive"
+	caseSensitive: Boolean
+
+  "Value to filter if it is Date input"
+  dateValue: DateTime
+
+  "Value to filter if it is Float Array input" 
+  floatArrayValue: [Float]
+
+  "Value to filter if it is Float input"
+  floatValue: Float
+
+  "Value to filter if it is Int Array input" 
+  intArrayValue: [Int]
+
+  "Value to filter if it is Int input"
+  intValue: Int
+
+  "Field name"
+	key : String!
+
+  "Logical NOT operator to negate the condition"
+  logNOT: Boolean
+
+  "Relational Operator to join the key and value"  
+	relOper: RelOpTypes!
```

Comment:
More inconsistent indentation

Reply:
There was tab instead of space, updated.